distribution/docs/src/main/resources/_contents/_configuring/configuring-product-cache-contents.adoc

```diff
@@ -0,0 +1,20 @@
+
+==== Configuring the Product Cache
+
+All caching properties are part of the <<ddf.catalog.resource.download.ReliableResourceDownloadManager,Resource Download Settings>>.
+
+===== Invalidating the Product Cache
+
+. The product cache directory can be administratively invalidated by turning off the product caching using the cacheEnabled property.
+. Alternatively, an administrator may manually invalidate products by removing them from the file system. Products are cached at the directory specified in the `productCacheDirectory` property. The following example assumes the
+`productCacheDirectory` has the default value of `<INSTALL-DIR>/data/product-cache`
+
+Format:
+
+`<INSTALL-DIR>/data/product-cache/<source-id>-<metacard-id>`
+
+Example:
+
+`<INSTALL-DIR>/data/product-cache/abc123`
+
+. Set Max Caching Directory Size. The `cacheDirMaxSizeMegabytes` property can be used as a way to evict the oldest products from the cache. By setting this to a low limit, the oldest products in the cache will be removed as new products are placed in the cache to ensure the cache does not go over the max limit.
```

Comment:
We're missing a _ton_ of configuration parameters for most features, so we may want to the be consistent and either _not_ document any of them and rely on the admin console, or make sure we document _all_ or them. If we go with the latter, we need to consider the admin console, command console and configuration files, which would be a _lot_ of work. Wonder if we could automate that and extract the information from some of the metadata we have.